corehq/apps/users/models.py

```diff
@@ -1596,7 +1596,7 @@ def can_access_any_web_apps(self, domain=None):
         if domain:
             try:
                 role = self.get_role(domain)
-                return role.permissions.web_apps_list
+                return bool(role.permissions.web_apps_list)
```

Comment:
Is this change required for the fix, or is it cleanup?

Reply:
just cleanup - I was reading this function to see what it did and I noticed this - shouldn't have any real effect though